<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 197, 255);</string>
  </property>
  <widget class="QPushButton" name="B_Sair">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_Conf_Conta">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string>Configurações de conta</string>
   </property>
  </widget>
  <widget class="QLabel" name="Img_User">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/USUARIO.png);
background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Back_Lista">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>231</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="B_MeusObjetivos">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Meus Objetivos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 255, 253);</string>
   </property>
   <property name="text">
    <string>Meus Objetivos</string>
   </property>
  </widget>
  <widget class="QLabel" name="NomeUsuario">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 229, 231);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_NovoObjetivo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Meus Objetivos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 255, 253);</string>
   </property>
   <property name="text">
    <string>Novo Objetivo</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_dadosConta">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; font-style:italic;&quot;&gt;Dados da conta&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_email">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Email:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_nome">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Nome:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_senha">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Senha:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_editar">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(170, 197, 255);</string>
   </property>
   <property name="text">
    <string>Editar/excluir</string>
   </property>
  </widget>
  <widget class="QLabel" name="C_email">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="C_nome">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="C_senha">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <zorder>Back_Lista</zorder>
  <zorder>B_Sair</zorder>
  <zorder>B_Conf_Conta</zorder>
  <zorder>B_MeusObjetivos</zorder>
  <zorder>NomeUsuario</zorder>
  <zorder>B_NovoObjetivo</zorder>
  <zorder>Img_User</zorder>
  <zorder>L_dadosConta</zorder>
  <zorder>L_email</zorder>
  <zorder>L_nome</zorder>
  <zorder>L_senha</zorder>
  <zorder>B_editar</zorder>
  <zorder>C_email</zorder>
  <zorder>C_nome</zorder>
  <zorder>C_senha</zorder>
 </widget>
 <resources>
  <include location="sourcee.qrc"/>
 </resources>
 <connections/>
</ui>
